econet_hvac_odu.yaml

```diff
@@ -0,0 +1,64 @@
+---

```

Comment:
> 1:4 [new-lines] wrong new line character: expected \n
Can you convert this to using UNIX new lines? If you can run pre-commit it should take it care for you. Probably easier to open this file in Notepad++ right click at the bottom right corner where it says "Windows (CR LF)" and change it to "Unix (LF)"